webpack.config.js

```diff
@@ -50,6 +50,10 @@ module.exports = {
         },
       }),
     ],
+    splitChunks: {
+      chunks: 'async',
+      name: (_, chunks) => chunks[0].name,
```

Comment:
Not totally sure about this `chunks[0]`, but it seemed the only way to get a chunk named `highlight.js`.